fix_distribution_policy.py

```diff
@@ -237,13 +232,15 @@ def alter(sqls, idx, nproc, dbname, port, host, user):
     parser_run = subparsers.add_parser('run', help='run the alter table cmds')
 
     parser_gen.add_argument('--out', type=str, help='outfile path for the alter table commands')
+    parser_gen.add_argument('--dump_legacy_ops', action='store_true', help='dump all tables with legacy distkey ops')
+    parser_gen.set_defaults(dump_legacy_ops=False)
```

Comment:
I guess most use cases are dumping the legacy ones. So maybe make that as the default?